CONTRIBUTING.md

```diff
@@ -124,13 +127,24 @@
 
 ## Python
 
+### Sphinx comments
+
+* Cantera Python documentation is based on the Python documentation generator
+  [Sphinx](https://www.sphinx-doc.org/en/master/index.html)
+* All classes, member variables, and methods should include
+  [Python docstrings](https://peps.python.org/pep-0257/#what-is-a-docstring)
+* Docstrings should use annotations compatible with
+  [automatic documentation generation from code](https://www.sphinx-doc.org/en/master/tutorial/automatic-doc-generation.html)
```

Comment:
Neither of these links explain how to add a Sphinx-parseable docstring to a member variable

Reply:
👍 ... the documentation of the `autodoc` extension is actually pretty poor, but there are other resources.